CONTRIBUTING.md

```diff
@@ -143,58 +144,57 @@ Make a PR modifying the following files with the right version:
 export const PACKAGE_VERSION = 'X.X.X'
 ```
 
+#### Github publish
 Once the changes are merged on `main`, you can publish the current draft release via the [GitHub interface](https://github.com/meilisearch/instant-meilisearch/releases): on this page, click on `Edit` (related to the draft release) > update the description (be sure you apply [these recommendations](https://github.com/meilisearch/integration-guides/blob/main/resources/integration-release.md#writting-the-release-description)) > when you are ready, click on `Publish release`.
 
 GitHub Actions will be triggered and push the package to [npm](https://www.npmjs.com/package/@meilisearch/instant-meilisearch).
 
+#### Codesandbox update
 Once the version is available on npm, please update the instant-meilisearch version used in the different Code-Sandboxes we provide:
 
 - [Meilisearch + InstantSearch](https://codesandbox.io/s/ms-is-mese9)
-- [Meilisearch + Vue InstantSearch](https://codesandbox.io/s/ms-vue-is-1d6bi)
+- [Meilisearch + Vue 2 InstantSearch](https://codesandbox.io/s/ms-vue-is-1d6bi)
+- [Meilisearch + Vue 3 InstantSearch](https://codesandbox.io/s/ms-vue3-is-0293zk)
 - [Meilisearch + React InstantSearch](https://codesandbox.io/s/ms-react-is-sh9ud)
 
 If you don't have the access to do it, please request it internally.
 
 #### Release a `beta` Version
 
-Here are the steps to release a beta version of this package:
+This package is able to create multiple types of betas:
+- A standard package beta, working on the latest version of Meilisearch.
```

Comment:
```suggestion
- A standard package beta, so a instant-meilisearch beta, compatible with the latest version of Meilisearch.
```

Reply:
> A standard npm package beta, so an Instant-Meilisearch beta, compatible with the latest version of Meilisearch.

These are all going to be published on `npm` with the beta tag. Do you think we should continue referring to instant-meilisearch as `instant-meilisearch` and not `Instant-Meilisearch` as it is the name of the took?

Reply:
> Unfortunately, these are all going to be published on npm with the beta tag :/

Oh yes you are right for the npm

> Do you think we should continue referring to instant-meilisearch as instant-meilisearch and not Instant-Meilisearch as it is the name of the took?

At your convenience! Sorry I don't remember the common guidelines for it